runtime-light/stdlib/file/file-system-functions.cpp

```diff
@@ -50,7 +52,17 @@ Optional<string> f$file_get_contents(const string &stream) noexcept {
 }
 
 task_t<Optional<int64_t>> f$fwrite(resource stream, string text) noexcept {
-  auto rsrc{from_mixed<class_instance<underlying_resource_t>>(stream, {})};
```

Comment:
I insist that this line should not change. This change makes me think that there is an error somewhere in the design of this PR. The main candidate for such an error for me is `mixed STDERR` declared in `FileSystemInstanceState`